kroxylicious-runtime/src/test/java/io/kroxylicious/proxy/internal/filter/FlakyFactory.java

```diff
@@ -11,6 +11,7 @@
 import org.apache.kafka.common.message.RequestHeaderData;
 import org.apache.kafka.common.protocol.ApiKeys;
 import org.apache.kafka.common.protocol.ApiMessage;
+import org.junit.jupiter.api.Assertions;
```

Comment:
We generally use the assertJ assertions, though I don't suppose it really matters in this context.

Reply:
That being the case, a very low priority task would be to update `FilterChainFactoryTest` to adhere to that norm.